ui/src/utils/getFilteredRulesInfo.js

```diff
@@ -0,0 +1,33 @@
+// This utility builds the string shown in the
+// info alert when no rules match the selected filters
+export default (productChannelQueries, rewindDate, onlyScheduledChanges) => {
+  let info = 'No rules found ';
```

Comment:
praise: This function seems to handle all states that the Rules page could be in cleanly and efficiently - nice work!